src/Symfony/Component/Validator/Tests/Validator/RecursiveValidator2Dot5ApiTest.php

```diff
@@ -29,4 +30,28 @@ protected function createValidator(MetadataFactoryInterface $metadataFactory, ar
 
         return new RecursiveValidator($contextFactory, $metadataFactory, $validatorFactory, $objectInitializers);
     }
+
+    public function testEmptyGroupsArrayDoesNotTriggerDeprecation()
```

Comment:
A, got it. :) There's no need to keep it in 3.0. However, the issue is that we cannot mark this as a legacy test because it wouldn't fail then if the method triggered a deprecation warning (unless I do miss anything).